website/content/docs/deprecation/index.mdx

```diff
@@ -10,6 +10,60 @@ description: |-
 
 This announcement page is maintained and updated periodically to communicate important decisions made concerning End of Support(EoS) for Vault features as well as features we have removed or disabled from the product. We document the removal of features, enable the community with a plan and timeline for eventual deprecations, and supply alternative paths to explore and evaluate to minimize business disruptions. If you have questions or concerns about a deprecated feature, please create a topic on [the community forum](https://discuss.hashicorp.com/c/vault/30) or raise a ticket with your support team. Please refer to the [FAQ](/docs/deprecation/faq) page for frequently asked questions concerning Vault feature deprecations.
 
+## Deprecation Status
```

Comment:
Thanks for updating the wording. I think the concept of mount is still probably confusing but I imagine it's a point of confusion across our docs, so I dunno how important it is to tidy it here.

I think this section should probably either go on the FAQ page or _under_ the deprecation table on this page, but open to other's opinions on this, and ok with the wording, so I'm going to approve it.